testing/test-helper.js

```diff
@@ -210,9 +210,9 @@ export class RequestBank {
 }
 
 export class BrowserController {
-  constructor(win) {
+  constructor(win, opt_rootNode) {
     this.win_ = win;
-    this.doc_ = this.win_.document;
+    this.doc_ = opt_rootNode || this.win_.document;
```

Comment:
This should also be renamed to `this.rootNode_`. You can still have `this.doc_ = this.rootNode_.ownerDocument || this.rootNode_` or something like this.